### DISCUSSION TITLE
Per-user hide & new items

### DISCUSSION THREAD

**Discussion by USER19464727** [elchininet/custom-sidebar#150] [Q&A] [answered]
I want to have a bunch of items in the sidebar for me but the other users want a much simpler experience. However, I want to add new_items and not hide things, which I have to hide for them. So far the closest thing I can do is set the global settings to basically hide everything for the other users and set myself as an exception to show everything that is default. However, this does not achieve exactly what I want (new items and such).

**Comment by elchininet** [comment]
Hi @USER19464727,
I don't undertand very well what your issue is, please, try to explain it concisely and it would be great of you can accompain the descriptions with images and code example of the configuration.
Regards

**Comment by USER19464727** [comment]
Basically imagine I would like 3 users each with different items hidden and some have new_items, whereas others don't have new_items. I can't tell how to do this. It seems as if there's 1 global setup and then possible exceptions, but only for a different order (not for hiding or new items).

**Comment by elchininet** [comment, marked as answer]
Hi @USER19464727,

The order property, available in the base config or in the exceptions, is an array of items and an item has many configuration options, among them "hide" and "new_item".

You can create a main config (the one that will be applied if no exception is matched) and different exceptions for different users, each one with different items and each item with `hide` or `new_item` options depending on your needs.

Have you tried this already?

**Reply by USER19464727** [reply]
Thanks! I misunderstood that _all_ order item properties were available under exceptions as well. I have it working. Including something like this in your example might help others noobs like me. Also, the example has “base_order” but the documentation says “extend_from_base”—was the name updated but not the example? Thanks for your help.

**Reply by elchininet** [reply]
>Thanks! I misunderstood that all order item properties were available under exceptions as well. I have it working. 

Glad that you have it working already 👍

>Including something like this in your example might help others noobs like me.

I will try to add some items to the examples.

>Also, the example has “base_order” but the documentation says “extend_from_base”—was the name updated but not the example?

Yes, that is. The `extend_from_base` is the new property since [v5.0.0](https://github.com/elchininet/custom-sidebar/releases/tag/v5.0.0). It was called `base_order` before. I'll change it.

>Thanks for your help.

You're welcome!

**Reply by elchininet** [reply]
Here: https://github.com/elchininet/custom-sidebar/pull/151
